### PR TITLE
Added state.vertices, state.edges, etc.

### PR DESCRIPTION
**Pull request by bramrodenburg**
**Description**
Added state properties for easily accessing `vertices`, `edges`, `graph_attributes`,  `edge_attributes` and `segmentations` in the Python Remote API. 

You can now do things such as:

```python
g = lk.createExampleGraph()
g.vertices
g.edges
g.edge_attributes
g.graph_attributes

g2 = g.findConnectedComponents(name="example")
g.segmentations["example"]
```

### CODE REVIEW

File: python/remote_api/src/lynx/kite.py [hunk 1]
```diff
@@ -884,6 +884,40 @@ def f(**kwargs):
...
+    return self._select_all('edge_attributes')
+
+  @property
+  def segmentations(self):
+    '''Access a given segmentation.
+
+    Example usage:
+    ```
+    graph = lk.createExampleGraph().findConnectedComponents(name='seg1')
+    segmentation = graph.segmentations['seg1']
```
Comment (darabos) [inline python/remote_api/src/lynx/kite.py:914]: I think this syntax makes it look like `graph.segmentations` is a dict. So a user may try `for s in graph.segmentations` or stuff like that which won't work. For this reason I think a simple function call is more honest and not any more verbose.

```suggestion
    segmentation = graph.segmentation('seg1')
```

File: python/remote_api/src/lynx/kite.py [hunk 2]
```diff
@@ -884,6 +884,40 @@ def f(**kwargs):
...
+    return self._select_all('graph_attributes')
+
+  @property
+  def edge_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the edge attributes as a table.'''
+    return self._select_all('edge_attributes')
+
+  @property
+  def segmentations(self):
+    '''Access a given segmentation.
```
Comment (darabos) [inline python/remote_api/src/lynx/kite.py:909]: Maybe mention that it uses a _Use segmentation as base graph_ box. That would clarify what kind of access it gives. (E.g. no belongs-to edges.)

File: python/remote_api/src/lynx/kite.py (hunk 1 above)
Comment (bramrodenburg) [reply python/remote_api/src/lynx/kite.py]: Done. Is this approximately what you had in mind?

File: python/remote_api/src/lynx/kite.py (hunk 2 above)
Comment (bramrodenburg) [reply python/remote_api/src/lynx/kite.py]: Given that I changed the `graph.segmentations` to a function call that now contains a one-liner that shows the `takeSegmentationAsBaseGraph` method, I think this is not really required anymore?

File: python/remote_api/src/lynx/kite.py [hunk 3]
```diff
@@ -884,6 +884,39 @@ def f(**kwargs):
...
+  def graph_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the graph attributes as a table.'''
+    return self._select_all('graph_attributes')
+
+  @property
+  def edge_attributes(self) -> 'SingleOutputAtomicBox':
+    '''Returns the edge attributes as a table.'''
+    return self._select_all('edge_attributes')
+
+  def segmentations(self, name: str) -> 'SingleOutputAtomicBox':
```
Comment (darabos) [inline python/remote_api/src/lynx/kite.py:907]: Sorry, just minor tweaks. I'd do them but I'm not sure if I can commit to your fork.
```suggestion
  def segmentation(self, name: str) -> 'SingleOutputAtomicBox':
```

It returns a single segmentation so let's call it `segmentation()`.

File: python/remote_api/src/lynx/kite.py [hunk 4]
```diff
@@ -884,6 +884,39 @@ def f(**kwargs):
...
+    '''Access a given segmentation by name.
```
Comment (darabos) [inline python/remote_api/src/lynx/kite.py:908]:
```suggestion
    '''Returns the named segmentation as a base project.
```

It's concise but specific enough to hint at the box. What do you think?

File: python/remote_api/src/lynx/kite.py (hunk 2 above)
Comment (darabos) [reply python/remote_api/src/lynx/kite.py]: > Given that I changed the `graph.segmentations` to a function call that now contains a one-liner that shows the `takeSegmentationAsBaseGraph` method, I think this is not really required anymore?

The documentation shouldn't assume that the user has read the source code. 😄

File: python/remote_api/src/lynx/kite.py (hunk 4 above)
Comment (bramrodenburg) [reply python/remote_api/src/lynx/kite.py]: I agree! Changed it.

File: python/remote_api/src/lynx/kite.py (hunk 3 above)
Comment (bramrodenburg) [reply python/remote_api/src/lynx/kite.py]: Done.

**Comment by darabos** [comment]
Not sure what's wrong with the frontend test but I'll look at it later. Thanks!

**Pull request merged**